wrappers/python/pybackend.cpp

```diff
@@ -187,6 +187,7 @@ PYBIND11_MODULE(NAME, m) {
         .value("enable_ir_reflectivity", RS2_OPTION_ENABLE_IR_REFLECTIVITY)
         .value("auto_exposure_limit", RS2_OPTION_AUTO_EXPOSURE_LIMIT)
         .value("auto_gain_limit", RS2_OPTION_AUTO_GAIN_LIMIT)
+        .value("rx_sensitivity", RS2_OPTION_RX_SENSITIVITY)
```

Comment:
I think you're missing the C# and other wrappers?

Reply:
will do when I will have the exact name of the control.